src/components/Divider/index.css.ts

```diff
@@ -0,0 +1,32 @@
+import { recipe } from '@vanilla-extract/recipes';
+import { vars } from '@/styles/theme.css';
+
+export const divider = recipe({
```

Comment:
컨벤션에 맞추어서 다음과 같이 네이밍 수정해주세요.
```suggestion
export const dividerStyle = recipe({
```

Reply:
반영완료했습니다!